README.md

```diff
@@ -34,6 +34,17 @@ DATABASE_URL=postgres://jrob@localhost:5432/floworky
 * Assertion Library: [Chai](http://chaijs.com/)
 * Test Spies, Stubs and Mocks: [Sinon](http://sinonjs.org/)
 * Test Coverage: [Istanbul](https://github.com/gotwarlost/istanbul)
+* Static Type Checking: [Flow](https://flowtype.org)
+
+### Flow
```

Comment:
Let's move this section into `CONTRIBUTING.md`, and condense it to a bullet point under `Quality Requirements`: `Flow type annotations are included`

Since it's listed under tech, I'm disinclined to add this information here after seeing how it increases the size of the README